projects/serializers.py

```diff
@@ -20,7 +22,7 @@ class Meta:
             "location",
             "catalog",
         )
-        extra_kwargs = {"creator": {"read_only": True}}
+        read_only_fields = ("id", "creator", "catalog", )
```

Comment:
Let me know if this is something that's not possible. At the moment, `Project.save` reassigns catalog based on impact level and catalog version, so passing a catalog id wasn't actually doing anything anyways.

Reply:
This makes sense to me but I'd rely on Tom's input.

Reply:
@TMCamp thoughts?

Reply:
Yep, that makes sense, @jwhite3.